src/huggingface_hub/inference/_client.py

```diff
@@ -1881,6 +1886,24 @@ def _resolve_url(self, model: Optional[str] = None, task: Optional[str] = None)
             else f"{INFERENCE_ENDPOINT}/models/{model}"
         )
 
+    @staticmethod
+    def get_recommended_model(task) -> str:
+        """
+        Get the model Hugging Face recommends for the input task.
+
+        Args:
+            task (`str`):
+                The Hugging Face task to get which model Hugging Face recommends.
+                All available tasks can be found [here](https://huggingface.co/tasks).
+
+        Returns:
+            `str`: Name of the model recommended for the input task.
+
+        Raises:
+            `ValueError`: If Hugging Face has no recommendation for the input task.
+        """
+        return _get_recommended_model(task)
```

Comment:
You can even get rid of the "private" `_get_recommended_model` and put the logic in `get_recommended_model` directly.

Reply:
Great suggestion, done!